challenges/CC_29_SmartRockets/dna.js

```diff
@@ -4,31 +4,43 @@
 // Code for: https://youtu.be/bGz7mv2vD6g
 
 function DNA(genes) {
+  //recieves genes and create a dna object 
   if (genes) {
     this.genes = genes;
-  } else {
+  } 
+  //if no genes just create random dna
+  else {
     this.genes = [];
     for (var i = 0; i < lifespan; i++) {
+      //gives random vectors
       this.genes[i] = p5.Vector.random2D();
+      //sets maximum force of vector to be applied to a rocket
       this.genes[i].setMag(maxforce);
     }
   }
-
+  // decides childs fitness
```

Comment:
This is not related to fitness but rather creates a new child based on two parent DNA objects.